sdk/js/src/Request.ts

```diff
@@ -134,8 +137,13 @@ export const publishEvents = async (
     const payload = generateEventPayload(config, user, events)
     logger.info(`Submit Events Payload: ${JSON.stringify(payload)}`)
 
+    let url = `${options?.apiProxyURL || EVENT_URL}${EVENTS_PATH}`
+    if (options?.enableObfuscation) {
+        url += '?obfuscated=1'
```

Comment:
should you use URLSearchParams like above?

Reply:
seemed like overkill since this is the only query param